.travis.yml

```diff
@@ -12,39 +12,39 @@ addons:
     packages: &BASE_PACKAGES
       # Build dependencies.
       # See also `apt-cache showsrc awesome | grep -E '^(Version|Build-Depends)'`.
-      - libnotify-bin
+      - imagemagick
       - libcairo2-dev
-      - gir1.2-gtk-3.0
+      - libdbus-1-dev
+      - libnotify-bin
       - libpango1.0-dev
-      - libxcb-xtest0-dev
+      - libstartup-notification0-dev
+      - libx11-xcb-dev
+      - libxcb-cursor-dev
       - libxcb-icccm4-dev
-      - libxcb-randr0-dev
       - libxcb-keysyms1-dev
-      - libxcb-xinerama0-dev
-      - libdbus-1-dev
-      - libxdg-basedir-dev
-      - libstartup-notification0-dev
-      - imagemagick
-      - libxcb1-dev
+      - libxcb-randr0-dev
       - libxcb-shape0-dev
       - libxcb-util0-dev
-      - libx11-xcb-dev
-      - libxcb-cursor-dev
-      - libxcb-xkb-dev
       - libxcb-xfixes0-dev
+      - libxcb-xinerama0-dev
+      - libxcb-xkb-dev
+      - libxcb-xtest0-dev
+      - libxcb1-dev
+      - libxdg-basedir-dev
       - libxkbcommon-dev
       - libxkbcommon-x11-dev
+      # Need xorg-macros
```

Comment:
for?

Reply:
Moved comment, but amended.  It is for building xcb-util-xrm (IIRC).